tools/src/addr-convert/Main.hs

```diff
@@ -30,7 +34,12 @@ optionsParser = do
         <> long    "address"
         <> help    "Address to convert. It must be in base64(url) format."
         <> metavar "STRING"
-    return AddrConvertOptions{..}
+    pm      <- textOption $
+           short   'p'
+        <> long    "protocolMagic"
+        <> help    "Generate mainnet or testnet address."
+        <> metavar "STRING"
```

Comment:
Are `mainnet` or `testnet` the two viable values here? If so, should the docs indicate this.

Reply:
Done!